### PR TITLE
SecureDrop Client 0.2.1

### PR DESCRIPTION
**Pull request by emkll**
# Description

Towards #1125 .

Related packaging PR: https://github.com/freedomofpress/securedrop-debian-packaging/pull/178

# Test Plan
- [x] Changelog is accurate
- [x] Version strings are correctly updated with `update_version.sh`

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.2.1
+
+* Delete source widgets for current sync update (#1114)
+* Reduce minimum window size (#1103)
+* Make sure source fingerprint is updated on sync (#1092)
```
Comment (eloquence) [inline changelog.md:7]: #974 was merged and not released since 0.2.0.

**Comment by eloquence** [comment]
Hm, there may have been some discussion that I missed - did we agree to exclude changes that aren't user-facing from the `changelog.md`? That would be a change compared with e.g. the 0.2.0 log. I personally think it's fine to err on being more comprehensive in the GitHub changelogs, and pare it down for other artifacts for public consumption (blog posts etc.).

**Comment by creviera** [comment]
I think it makes sense to skip refactors, like moving code from one place to another, running black/isort, or just adding a bunch of tests, which is all that's left over here. But I have no objections if we want to include the rest of the changes since we released 0.2.0, which are (see https://github.com/freedomofpress/securedrop-client/compare/0.2.0...main):
* Move CSS strings to CSS files and add new CSS tests: https://github.com/freedomofpress/securedrop-client/pull/1082
* Use QLabel instead of QWidget for journalist designation: https://github.com/freedomofpress/securedrop-client/pull/1112
* Reformat the code with black and isort: https://github.com/freedomofpress/securedrop-client/pull/1115

**Comment by eloquence** [comment]
There's also #974 which I think is significant (functional change, with user-facing implications - syncs may be slowed down by duplicate jobs - but no UI impact). For these two:
> * Move CSS strings to CSS files and add new CSS tests: https://github.com/freedomofpress/securedrop-client/pull/1082
> * Reformat the code with black and isort: https://github.com/freedomofpress/securedrop-client/pull/1115
In SD Core we would probably group this under "Developer Workflow" since it does not change functionality.

File: changelog.md (hunk 1 above)
Comment (eloquence) [reply changelog.md]: (That was meant to say #975, sorry for the confusion)

**Comment by emkll** [comment]
Thanks for the reviews ! CI is passing on the new tag, merging! (see https://github.com/freedomofpress/securedrop-debian-packaging/pull/178)

**Pull request merged**